math/mathcore/inc/Fit/SparseData.h

```diff
@@ -23,42 +24,56 @@ namespace ROOT {
       // This is a proxy to a std::list<Box>
       class ProxyListBox;
 
+      /**
+         SparseData class representing the data of a THNSparse histogram
+         The data needs to be converted to a BinData class before fitting using
+         the GetBinData functions.
+
+         @ingroup FitData
+      */
+
       class SparseData : public FitData  {
       public:
-         //Constructor with a vector
+         /// Constructor with a vector
          SparseData(std::vector<double>& min, std::vector<double>& max);
 
-         //Constructor with a dimension and two arrays
+         /// Constructor with a dimension and two arrays
          SparseData(const unsigned int dim, double min[], double max[]);
 
-         //Destructor
+         /// Copy constructor
+         SparseData(const SparseData & rhs);
+
+         /// Destructor
```

Comment:
```suggestion
         /// Destructor
```
The destructor is now empty, you could get rid of it

Reply:
Actually this is needed in order to forward declare ProxyListBox.

Reply:
Okay!